### ISSUE TITLE
Beta Version 1.6.0 Testing

### ISSUE THREAD

**Issue by donavanbecker** [homebridge-plugins/homebridge-platform-wemo#191]
To Test:
```
sudo npm install -g --unsafe-perm homebridge-platform-wemo@beta
```
To Revert:
```
sudo npm install -g --unsafe-perm homebridge-platform-wemo@latest
```

**Comment by donavanbecker** [comment]
# Dimmer Night mode: Poll brightness when turning the lights on


This allows night mode to function with homebridge knowing correct night mode brightness value when lights turn on. (Previously if dimmer was set to 80% normal and 20% night mode, at night when turning on dimmers, Home app would report 80% brightness when in fact the dimmer is 20% brightness. This fixes and now correctly shows 20% brightness.)

**Comment by donavanbecker** [comment]
@leeliu can you test the beta to make sure it is working?

**Comment by leeliu** [comment]
Apologies...just confirmed it's working well on beta. Though it says 1.5.7 (but i noticed the version wasn't bumped, so that makes sense).

**Comment by donavanbecker** [comment]
Awesome! I will release 1.6.0!

@leeliu Are you interested in taking over this repo?

**Comment by donavanbecker** [comment]
@leeliu just released 1.6.0!

**Comment by leeliu** [comment]
@donavanbecker Thanks for the offer! I’m afraid I can’t really commit the time. It is a great project though and I do plan to make future contributions when I can.
